tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -158,16 +157,10 @@ <h3>No profile data was found.</h3>
         value: 0,
         observer: '_selectedToolChanged'
       },
-      // This is combination of selectedDataset and selectedTool.
-      // Most significant bits contains index of datasets;
-      // Lest Significant bits contains index of tools;
-      // With this arrangement, we can achieve cascading Polymer effects
-      // propagation and atomic change of both dataset and tool.(e.g.
-      // when dataset change, the tool set to the first available one).
       currentSelected: {
```

Comment:
Could you add a comment noting why this object exists? Similar to the previous comment.